### PR TITLE
Add group

### PR DESCRIPTION
**Pull request by TiboldiRoberto**
What does it fix?

Add: 
- AddGroupNameView
- SetMeetPointsView
- GroupView

How has it been tested?

### CODE REVIEW

File: DeUrgenta/Features/CreateGroup/View/AddGroupNameView.swift [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                TextField(AppStrings.AddGroupName.textFieldPlaceholder.localized(), text: $groupName)
+                    .textFieldStyle(.roundedBorder)
+            }
+            .padding()
+
+            Spacer()
+
+            VStack {
+                Button(action: {
+                    delegate.addGroupNameViewDidTapContinue(self)
```
Comment (MihaiPantiru) [inline DeUrgenta/Features/CreateGroup/View/AddGroupNameView.swift]: Let's add a rule to make sure we have a group name set before allowing to move to the next step.
1) continue with no text - display alert - Denumirea grupului este obligatorie.
2) continue with text - all good

**Review by MihaiPantiru — CHANGES_REQUESTED** [review]
Awesome work 🙌 I added just a small change request - to validate we have a group name filled before allowing the user to move to the next step.

File: DeUrgenta/Features/CreateGroup/View/AddGroupNameView.swift (hunk 1 above)
Comment (TiboldiRoberto) [reply DeUrgenta/Features/CreateGroup/View/AddGroupNameView.swift]: Ok

File: DeUrgenta/Features/CreateGroup/View/AddGroupNameView.swift [hunk 2]
```diff
@@ -48,6 +54,9 @@ struct AddGroupNameView: View {
                     .cornerRadius(6)
                 })
             }
+            .alert(isPresented: $showingAlert, content: {
+                Alert(title: Text("Important"), message: Text("Denumirea grupului este obligatorie."), dismissButton: .default(Text("OK")))
```
Comment (MihaiPantiru) [inline DeUrgenta/Features/CreateGroup/View/AddGroupNameView.swift]: Can we localize the alert text, as we have for other strings?

Comment (TiboldiRoberto) [reply DeUrgenta/Features/CreateGroup/View/AddGroupNameView.swift]: Yes, of course :)

**Review by MihaiPantiru — APPROVED** [review]

**Pull request merged**